libcontainer/cgroups/fs/utils.go

```diff
@@ -55,12 +55,17 @@ func getCgroupParamKeyValue(t string) (string, uint64, error) {
 
 // Gets a single uint64 value from the specified cgroup file.
 func getCgroupParamUint(cgroupPath, cgroupFile string) (uint64, error) {
-	contents, err := ioutil.ReadFile(filepath.Join(cgroupPath, cgroupFile))
+	fileName := filepath.Join(cgroupPath, cgroupFile)
+	contents, err := ioutil.ReadFile(fileName)
 	if err != nil {
 		return 0, err
 	}
 
-	return parseUint(strings.TrimSpace(string(contents)), 10, 64)
+	res, err := parseUint(strings.TrimSpace(string(contents)), 10, 64)
+	if err != nil {
+		return res, fmt.Errorf("unable to parse %q as a uint from Cgroup file %q", string(contents), fileName)
```

Comment:
@hqhq to be honest, I didn't actually run into this error while testing, I was trying to track down another int parsing error and thought perhaps this was the one generating the error I saw, so I added the more detailed error string.  It turned out to be someplace else in the code but then, as you said, I thought it would be good to be more detailed "just in case".  If you think error case will never happen I'm ok with removing it.